banyand/measure/write.go

```diff
@@ -148,6 +149,14 @@ func (w *writeCallback) handle(dst map[string]*dataPointsInGroup, writeEvent *me
 			} else {
 				tagValue = tagFamily.Tags[j]
 			}
+			tagFamiliesForIndexWrite[i].values = append(tagFamiliesForIndexWrite[i].values, encodeTagValue(
```

Comment:
It is better to merge the encoding operation in lines 152 and 160 and encode the tag value only once.